builtin/providers/aws/provider.go

```diff
@@ -172,6 +172,7 @@ func Provider() terraform.ResourceProvider {
 			"aws_vpc":                      dataSourceAwsVpc(),
 			"aws_vpc_endpoint_service":     dataSourceAwsVpcEndpointService(),
 			"aws_vpc_peering_connection":   dataSourceAwsVpcPeeringConnection(),
+			"aws_kms_secrets":              dataSourceAwsKmsSecrets(),
```

Comment:
I went with the plural since you can have multiple secrets defined in the data source.  I'm happy to change it if the consensus is it should be singular.  @stack72 ?